packages/cli/src/Server.ts

```diff
@@ -1539,6 +1546,20 @@ class App {
 			),
 		);
 
+		// ----------------------------------------
+		// EventBus Setup
+		// ----------------------------------------
+
+		if (!eventBus.isInitialized) {
+			await eventBus.initialize();
+		}
+		// add Event Bus REST endpoints
+		this.app.use(`/${this.restEndpoint}/eventbus`, eventBusRouter);
+
+		if (process.env.NODE_ENV !== 'production') {
+			config.set('enterprise.features.logStreaming', true);
+		}
+
```

Comment:
Should we remove this before merging?

Reply:
we can. it just makes testing a bit harder.